apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/R4ExplanationOfBenefitResourceProvider.java

```diff
@@ -268,22 +269,34 @@ public ExplanationOfBenefit read(@IdParam IdType eobId, RequestDetails requestDe
   @Trace
   public Bundle findByPatient(
       @RequiredParam(name = ExplanationOfBenefit.SP_PATIENT)
-          @Description(shortDefinition = "The patient identifier to search for")
+          @Description(
+              shortDefinition = OpenAPIContentProvider.PATIENT_SP_RES_ID_SHORT,
+              value = OpenAPIContentProvider.PATIENT_SP_RES_ID_VALUE)
           ReferenceParam patient,
       @OptionalParam(name = "type")
-          @Description(shortDefinition = "A list of claim types to include")
+          @Description(
+              shortDefinition = OpenAPIContentProvider.EOB_CLAIM_TYPE_SHORT,
+              value = OpenAPIContentProvider.EOB_CLAIM_TYPE_VALUE)
           TokenAndListParam type,
       @OptionalParam(name = "startIndex")
-          @Description(shortDefinition = "The offset used for result pagination")
+          @Description(
+              shortDefinition = OpenAPIContentProvider.PATIENT_START_INDEX_SHORT,
+              value = OpenAPIContentProvider.PATIENT_START_INDEX_VALUE)
           String startIndex,
       @OptionalParam(name = "excludeSAMHSA")
-          @Description(shortDefinition = "If true, exclude all SAMHSA-related resources")
+          @Description(
+              shortDefinition = OpenAPIContentProvider.EOB_EXCLUDE_SAMSHA_SHORT,
+              value = OpenAPIContentProvider.EOB_EXCLUDE_SAMSHA_VALUE)
           String excludeSamhsa,
       @OptionalParam(name = "_lastUpdated")
-          @Description(shortDefinition = "Include resources last updated in the given range")
+          @Description(
+              shortDefinition = OpenAPIContentProvider.PATIENT_LAST_UPDATED_SHORT,
+              value = OpenAPIContentProvider.PATIENT_LAST_UPDATED_VALUE)
           DateRangeParam lastUpdated,
       @OptionalParam(name = "service-date")
-          @Description(shortDefinition = "Include resources that completed in the given range")
+          @Description(
+              shortDefinition = OpenAPIContentProvider.EOB_SERVICE_DATE_SHORT,
+              value = OpenAPIContentProvider.EOB_SERVICE_DATE_VALUE)
```

Comment:
EOB also has a read endpoint.; its a bit more complicated because many items for read are contained in the request details and not individual params. Would need to figure out how to handle that in OpenAPI.